generated/client/client.go

```diff
@@ -70,7 +70,10 @@ func (c Client) GetBooks(ctx context.Context, i *models.GetBooksInput) (models.G
 	path = path + "?" + urlVals.Encode()
 
 	client := &http.Client{Transport: c.transport}
-	req, _ := http.NewRequest("GET", path, bytes.NewBuffer(body))
+	req, err := http.NewRequest("GET", path, bytes.NewBuffer(body))
+	if err != nil {
+		return nil, err
```

Comment:
// TODO: decide how to represent this error (should it be `model.DefaultInternalError`?) and whether the client should retry